mscore/pianoroll/pianoroll.cpp

```diff
@@ -318,7 +318,7 @@ PianorollEditor::PianorollEditor(QWidget* parent)
 
       tbTweak->addWidget(new QLabel(tr("Len:")));
       tbTweak->addWidget((tickLen = new QSpinBox));
-      tickLen->setRange(-2000, 2000);
+      tickLen->setRange(-2000, 60000);
```

Comment:
probably should just be 0 to 60000?

Reply:
64000 would be ideal since a 1/64th note could then be made to sustain for a whole beat.